jenkinsapi_tests/unittests/test_crumbed_requester.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function
+import requests
```

Comment:
It's used for the patching of the [get](https://github.com/MichaelPereira/jenkinsapi/blob/f7a7c8f12ea200ac231dfe7135a9ef38e6587cea/jenkinsapi_tests/unittests/test_crumbed_requester.py#L13) and [post](https://github.com/MichaelPereira/jenkinsapi/blob/f7a7c8f12ea200ac231dfe7135a9ef38e6587cea/jenkinsapi_tests/unittests/test_crumbed_requester.py#L33) request methods for the purpose of the unit tests.

I don't see the problem with the unit tests and patching, they ensure that the functionality is  working in isolation of all other factors by making sure that the `get_url()` and `post_url()` methods of this class will retrieve crumb data before making the actual call.

There is already a [systest for the Crumb Requester](https://github.com/MichaelPereira/jenkinsapi/blob/f7a7c8f12ea200ac231dfe7135a9ef38e6587cea/jenkinsapi_tests/systests/test_crumbs_requester.py) but I don't think it's working properly because without the changes of this Pull Request, it should fail at [this line](https://github.com/MichaelPereira/jenkinsapi/blob/f7a7c8f12ea200ac231dfe7135a9ef38e6587cea/jenkinsapi_tests/systests/test_crumbs_requester.py#L92) since this would make a normal Requester `get_url()` call to a supposedly CSRF-security enabled Jenkins server. My internal Jenkins knowledge is not good enough to spot anything wrong in that systest, so I wouldn't know how to go about fixing it without more guidance.